Makefile.am

```diff
@@ -22,6 +22,7 @@ EXTRA_DIST = \
 	autogen.sh \
 	$(gsettings_SCHEMAS) \
 	com.inform7.IDE.desktop.in \
+	com.inform7.IDE.appdata.xml \
```

Comment:
Oh, yeah, I submitted this late at night and forgot I'd never figured out a mechanism for installing to the correct directory. :laughing: I implemented your suggestion above in the amended commit. I'm pretty unfamiliar with the way autotools works under the hood, so I trust it will work.